repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeFilter.scala

```diff
@@ -138,7 +138,7 @@ class ReposeFilter @Inject()(@Value(ReposeSpringProperties.NODE.NODE_ID) nodeId:
 
         MDC.put(TracingKey.TRACING_KEY, traceGUID)
 
-        val doFilterChain = TryWith(filterContextList) { _ =>
```

Comment:
Did we lose closing the `filterContextList`? I'm not seeing it in this revision.

Reply:
We really did. Somewhat concerning that a test didn't catch the problem. I'll see what i can do.

Reply:
One of those situations where you think really hard about not forgetting to add the thing when you are done, and then forget anyway